mlflow/R/mlflow/R/project-run.R

```diff
@@ -20,7 +20,7 @@
 #' @return The run associated with this run.
 #'
 #' @export
-mlflow_run <- function(entry_point = NULL, uri = ".", version = NULL, param_list = NULL,
+mlflow_run <- function(uri = ".", entry_point = NULL, version = NULL, parameters = NULL,
```

Comment:
Switched the order of `entry_point` and `uri` to match Python

Reply:
We'll want to make a note of this since it breaks `mlflow_run("stuff.R")` which is a common use case.

Reply:
Does it? I thought you can do 
```
mlflow run path/to/stuff.R

Reply:
^Yep, using the CLI should still work since it invokes `Rscript -e /path/to/script.R`, but yeah we need to document that users should change R code like `mlflow_run("/path/to/script.R)` to `mlflow_run(entry_point = "/path/to/script.R")` (e.g. for R users who are calling `mlflow_run` to run a single script in the current directory). Would mentioning this in release notes / the MLflow 1.0 migration guide suffice?

Reply:
I think that's ok, thanks for the clarification Sid!

Reply:
Yep that's exactly it. It might be worth an informative error message if the user passes a `.py` or `.R` file first. Do you guys think it makes sense to have this at the python level or just in the R function?

Reply:
I think we should have it at the python level. Hopefully it would propagate nicely back to R.